README.rst

```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
 ::
 
 $ python host_os.py build-iso --help
+
+How to contribute a new package to Host OS
+------------------------------------------
+
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory: 
+
+1. Create a metadata file <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is at https://github.com/open-power-host-os/versions/package_metadata_yaml_format.yaml
```

Comment:
As we discussed today, I'll rename the file to package_metadata_yaml_template.yaml . Rename it here